views/debug.hbs

```diff
@@ -0,0 +1,17 @@
+<h1> Debug </h1>
+
+<h2> Create User </h2>
```

Comment:
Could you move this to the users page? There should be a view for this

Reply:
Lemme delete this. Made it to test but didnt use it in the end